crates/test-case-utils/Cargo.toml

```diff
@@ -0,0 +1,29 @@
+[package]
+name          = "test-case-utils"
+version       = "2.2.2"
+edition       = "2018"
+authors       = ["Marcin Sas-Szymanski <marcin.sas-szymanski@anixe.pl>", "Wojciech Polak <frondeus@gmail.com>", "Łukasz Biel <lukasz.p.biel@gmail.com>"]
+description   = "Provides utils for parsing #[test_case(...)] procedural macro attribute for generating parametrized test cases easily"
```

Comment:
Feel free to tweak this - this Cargo.toml is mostly copy-paste from the old `test-case-macros/Cargo.toml` with a different description and dropping the `proc-macro = true`